src/codeu/chat/client/core/Context.java

```diff
@@ -26,7 +26,7 @@
 
 public final class Context {
 
-  private final BasicView view;
+  private final View view;
```

Comment:
Code is well-designed for the system (the code is broken up into classes appropriately)